### ISSUE TITLE
IsSafari is broken

### ISSUE THREAD

**Issue by oppai** [ICJIA/vue-browser-detect-plugin#31]
My iOS is 14.4
<img src="https://user-images.githubusercontent.com/1712116/112299166-e0f76180-8cda-11eb-9082-e49860998ed5.png" width=400px>

**Comment by cschweda** [comment]
Thanks. I need to add a key for Safari on iOS.

My plan is to update all the browser detection checks in the near future. That should fix some of the issues.

**Comment by trapcodeio** [comment]
Did this update happen? 😁  @cschweda

**Comment by JBtje** [comment]
I can confirm the problem still exists. @cschweda any plans on an update?

**Comment by cschweda** [comment]
yes -- need to update for Vue 3 and Nuxt 3. No timeframe yet -- but it's on my radar. 

PRs much appreciated.
